vagrant-kubernetes-istio/RunTestOnHost/README.md

```diff
@@ -67,6 +67,23 @@ cd $ISTIO/istio
 make e2e_simple E2E_ARGS="--use_local_cluster"
 ```
 
+### 3. Debug with Delve
+You can try debugging Istio with debugger tool. vm_setup.sh already installs Delve into VM for us. To use Delve, we need process id of the binary we want to debug.
+For example, if you want to run test in the VM and debug. You can issue this command to run e2e_simple test.
```

Comment:
How about something like this:

For example, If you want to debug a test, run that test in your host/vm.
```bash
In the VM/Host
make e2e_simple  E2E_ARGS="--use_local_cluster --skip_cleanup"
```

Now, login into the vm and find the pid of the process say pilot-discovery that you want to debug
```bash
vagrant ssh
ps -ef | grep pilot-discovery
```

Reply:
Thanks. I modified this part accordingly.